### PR TITLE
Maya 2022.1 has the Light loop API V2 and is compatible with MaterialX

### PR DESCRIPTION
**Pull request by JGamache-autodesk**
- Updated MaterialX build how-to.
- Transparency support
- Updated list of topo nodes

### CODE REVIEW

File: lib/mayaUsd/render/vp2RenderDelegate/material.cpp [hunk 1]
```diff
@@ -237,6 +237,24 @@ TF_DEFINE_PRIVATE_TOKENS(
...
+    (ND_convert_vector3_vector2)
+    (ND_convert_vector3_vector4)
+    (ND_convert_vector4_color4)
+    (ND_convert_vector4_vector3)
+    (ND_convert_color3_vector3)
+    (ND_convert_color4_vector4)
+    (ND_convert_color3_color4)
+    (ND_convert_color4_color3)
+    (ND_convert_boolean_float)
+    (ND_convert_integer_float)
```
Comment (JGamache-autodesk) [inline lib/mayaUsd/render/vp2RenderDelegate/material.cpp:257]: These nodes also affect topology, as found while trying the brick procedural MaterialX demo file.

File: lib/mayaUsd/render/vp2RenderDelegate/material.cpp [hunk 2]
```diff
@@ -1877,6 +1895,8 @@ MHWRender::MShaderInstance* HdVP2Material::_CreateMaterialXShaderInstance(
         // Add automatic tangent generation:
         shaderInstance->addInputFragment("materialXTw", "Tw", "Tw");
 
+        shaderInstance->setIsTransparent(ogsFragment.isTransparent());
```
Comment (JGamache-autodesk) [inline lib/mayaUsd/render/vp2RenderDelegate/material.cpp:1898]: This requires getting at least https://github.com/autodesk-forks/MaterialX/pull/1285 so the transparency is correctly computed.

Comment (kxl-adsk) [reply lib/mayaUsd/render/vp2RenderDelegate/material.cpp:1898]: It sounds like we should update the build instructions in the doc with the above info as well?

File: lib/mayaUsd/render/vp2RenderDelegate/material.cpp (hunk 1 above)
Comment (kxl-adsk) [reply lib/mayaUsd/render/vp2RenderDelegate/material.cpp:257]: where are these tokens use?

File: lib/mayaUsd/render/vp2RenderDelegate/material.cpp (hunk 2 above)
Comment (JGamache-autodesk) [reply lib/mayaUsd/render/vp2RenderDelegate/material.cpp:1836]: Done.

File: lib/mayaUsd/render/vp2RenderDelegate/material.cpp (hunk 1 above)
Comment (JGamache-autodesk) [reply lib/mayaUsd/render/vp2RenderDelegate/material.cpp]: They are used to recognize nodes that affect the topology of the shader graph when we compute the graph key for shader re-use. I used a MaterialX API to simplify the list in the latest commit.

**Review by kxl-adsk — APPROVED** [review]
Much cleaner with the switch to use category. LGTM.

**Pull request merged**